subgraphs/burrow/src/handlers/farm.ts

```diff
@@ -68,56 +62,49 @@ export function handleAddAssetFarmReward(event: EventData): void {
   }
 
   const reward_amount = data.get("reward_amount");
-  if (!reward_amount) {
-    log.warning("handleAddAssetFarmReward() :: reward_amount not found", []);
-    return;
-  }
-
   const rewardToken = getOrCreateRewardToken(reward_token, farmType);
 
   const market = getOrCreateMarket(farm);
-  const reward_tokens = market.rewardTokens!;
+  const rewardTokens = market.rewardTokens!;
   // push if reward token does not exists, get index
-  let reward_token_index = reward_tokens.indexOf(rewardToken.id);
+  let reward_token_index = rewardTokens.indexOf(rewardToken.id);
   if (reward_token_index == -1) {
-    reward_tokens.push(rewardToken.id);
-    market.rewardTokens = reward_tokens;
-    reward_token_index = reward_tokens.length - 1;
+    rewardTokens.push(rewardToken.id);
+    market.rewardTokens = rewardTokens;
+    reward_token_index = rewardTokens.length - 1;
   }
 
   // add reward_amount to _reward_remaining_amounts
-  const _reward_remaining_amounts = market._reward_remaining_amounts;
-  if (_reward_remaining_amounts.length <= reward_token_index) {
-    _reward_remaining_amounts.push(BigInt.fromString(reward_amount.toString()));
+  const _rewardRemainingAmounts = market._reward_remaining_amounts;
+  if (_rewardRemainingAmounts.length <= reward_token_index) {
+    _rewardRemainingAmounts.push(BigInt.fromString(reward_amount!.toString()));
   } else {
-    _reward_remaining_amounts[reward_token_index] =
+    _rewardRemainingAmounts[reward_token_index] =
       market._reward_remaining_amounts[reward_token_index].plus(
-        BigInt.fromString(reward_amount.toString())
+        BigInt.fromString(reward_amount!.toString())
       );
   }
-  market._reward_remaining_amounts = _reward_remaining_amounts;
+  market._reward_remaining_amounts = _rewardRemainingAmounts;
```

Comment:
Can you fix these field names in the schema: any odd on field or entity is just prefixed with an underscore. Should be `_rewardRemainingAmounts`

Reply:
I am asking for the schema fields to keep the same casing `_camelCase`.. https://github.com/prasad-kumkar/messari-subgraphs/blob/burrow/subgraphs/burrow/schema.graphql#L661